crates/tlog_tiles/src/tile.rs

```diff
@@ -41,18 +41,19 @@ const PATH_BASE: u64 = 1000;
 /// `Tile{H: 3, L: 4, N: 1234067, W: 8}`'s path is `tile/3/4/x001/x234/067`.  See the [`Tile::path`]
 /// method and the [`Tile::from_path`] function.
 ///
-/// The `is_data` field indicates that the tile holds raw record data instead of hashes.  In this
-/// case, the level is encoded in the tile path as the path element "data".
+/// The `data_elem` field indicates that the tile holds raw record data instead of hashes, and
+/// provides the path element ("data" for static-ct-api or "entries" for tlog-tiles) to use when
+/// encoding the level in the tile path.
 ///
 /// See also <https://golang.org/design/25530-sumdb#checksum-database> and
 /// <https://research.swtch.com/tlog#tiling_a_log>.
 #[derive(Debug, Eq, Hash, PartialEq, Default, Clone, Copy)]
 pub struct Tile {
-    h: u8,         // height of tile (1 ≤ H ≤ 30)
-    l: u8,         // level in tiling (0 ≤ L ≤ 63)
-    n: u64,        // number within level (0 ≤ N, unbounded)
-    w: u32,        // width of tile (1 ≤ W ≤ 2**H; 2**H is complete tile)
-    is_data: bool, // whether or not this is a data tile
+    h: u8,                               // height of tile (1 ≤ H ≤ 30)
+    l: u8,                               // level in tiling (0 ≤ L ≤ 63)
+    n: u64,                              // number within level (0 ≤ N, unbounded)
+    w: u32,                              // width of tile (1 ≤ W ≤ 2**H; 2**H is complete tile)
+    data_path_opt: Option<&'static str>, // whether or not this is a data tile, and the data path element to use for encoding
```

Comment:
Could this be an enum instead of a string constant? The benefit would be that the compiler would tell us if we accidentally used an unknown variant.

```rust
// not sure about the best name
pub enum TilePath{
  Data,
}

// perhaps we also need this:
impl AsRef<str> for TilePath {
  fn as_ref(&self) -> &str {
     match self {
        Self::Data => "data",
     }
  }
}
```

```suggestion
    data_path_opt: Option<TilePath>, // whether or not this is a data tile, and the data path element to use for encoding
```